conversion/conversion.go

```diff
@@ -587,6 +611,25 @@ func WriteSessionFile(conv *internal.Conv, name string, out *os.File) {
 	fmt.Fprintf(out, "Wrote session to file '%s'.\n", name)
 }
 
+// WriteDownloadableFiles creates a directory labeled downloads with the current timestamp
+// where it writes the sessionfile, report summary and DDLs then returns the directory where it writes.
+func WriteDownloadableFiles(conv *internal.Conv, dbName string, driver string, BytesRead int64, out *os.File) (string, error) {
```

Comment:
I can't think of a better name for this variable, but if you think of something that seems better, please feel free to update. DownloadFiles seems too generic.